starks/poly_utils.py

```diff
@@ -63,8 +63,94 @@ def construct_affine_vanishing_polynomial(field: Field, aff: AffineSpace) -> Pol
   """Constructs a polynomial which vanishes over a given affine space."""
   # TODO(rbharath): Need to implement this correctly.
   aff_elts = [field(elt) for elt in aff]
+  ##############################################
+  # TODO(rbharath): This doesn't work for arbitrary finite fields! 
+  print("field")
+  print(field)
+  print("aff_elts")
+  print(aff_elts)
+  print("[type(aff) for aff in aff_elts]")
+  print([type(aff) for aff in aff_elts])
+  # Ok the error here is that aff_elts are not being interpreted as finite field elements? How to fix?
+  ##############################################
+  print(aff_elts)
   return zpoly(field, aff_elts)
 
+def construct_affine_vanishing_polynomial_Moore(field: Field, aff: AffineSpace) -> Poly:
```

Comment:
Could we add a test that the vanishing polynomial works correctly? (That is, it takes `aff` to 0)